packages/libs/s3-static-assets/package.json

```diff
@@ -33,13 +33,12 @@
   "homepage": "https://github.com/serverless-nextjs/serverless-next.js#readme",
   "dependencies": {
     "aws-sdk": "2.728.0",
+    "fast-glob": "^3.2.5",
```

Comment:
Refactored from using `klaw` to `fast-glob` as CDK relies on synchronous operations during synthesis.